### ISSUE TITLE
Feature Request: Launch ANY App by Corner Gesture

### ISSUE THREAD

**Issue by adhishthite** [AChep/AcDisplay#87]
Hello !

Loved the 3.6 version.

I have a small feature request. Why not enable the launching of ANY user selected apps through corner gestures? Why only Dialer and Camera?
If you can include this, it'll really be helpful.

Thanks.

Also, a minor change:
When we select a gesture, the combo box that opens up shows:
'Launch a camera' and 'Launch a dialer'
The 'a' is not needed.

**Comment by hipio** [comment]
@AChep, AcDisplay is already my favorite lockscreen because I love the simplicity and minimality of it, but if you added this, it would be even more perfect!

**Comment by AChep** [comment]
@hipio yeah, this feature is in my _to-do_ list. However, https://github.com/AChep/AcDisplay/blob/master/project/app/src/main/java/com/achep/acdisplay/ui/CornerHelper.java is that class.

**Comment by hipio** [comment]
@AChep Thanks! I'll take a look this weekend (need to finish up some school work first) and see if I can implement it myself. I've added a $5 bounty to this suggestion as a thank you or motivation to work on this. :P

**Comment by hipio** [comment]
@AChep, where can I find the dialog that shows the possible options? I have added my option, changed the config, added my stuff to the CornerHelper and all I need to do now is let the user pick an app they want to launch and save that.

**Comment by AChep** [comment]
@hipio currently there's no custom dialog for that :) You'll have to create a one. Example: https://github.com/AChep/AcDisplay/tree/master/project/app/src/main/java/com/achep/acdisplay/ui/preferences

**Comment by hipio** [comment]
@AChep, well, I would if I knew what happens when you press one of the corner gestures. A dialog pops up and I need to know where it was created before I can change it. :P

**Comment by hipio** [comment]
@AChep Well, I got everything working, except that the icon of the app is totally black (it uses the PackageManager to extract the Drawable from a custom app, which I assign to mDrawable in CircleView in the updateIcon() function). Not sure where the Drawable is made black though. Must be an inversion somewhere as you use white icons.

**Comment by hipio** [comment]
@AChep Pull request created. https://github.com/AChep/AcDisplay/pull/93#partial-pull-merging

**Comment by GOGO98901** [comment]
Is there any update on this as it is a neat feature?

**Comment by hipio** [comment]
@AChep Any ETA on when my push (#93) is getting merged?

**Comment by hipio** [comment]
I'm going to assume that #93 this will not be merged.

**Comment by mjt01** [comment]
Just curious as to the status of this issue/pr? Seems dormant atm, but this would be a great feature.

**Comment by CarlosACepeda** [comment]
I'm taking care of this PR now.
